dap_prinz_green_jobs/pipeline/ojo_application/extract_green_skills_flow.py

```diff
@@ -0,0 +1,202 @@
+"""
+Extract green skills pipeline
+---------
+
+A pipeline that takes a sample of job adverts and extacts and maps
+skills onto a custom green skills taxonomy.
+
+(with default parameters):
+
+python dap_prinz_green_jobs/pipeline/ojo_application/extract_green_skills_flow.py run --extract_skills_library_path "PATH/TO/YOUR/LIBRARY-PACKAGE"
+"""
+from metaflow import FlowSpec, S3, step, Parameter
+from dap_prinz_green_jobs.getters.data_getters import (
+    get_s3_resource,
+    load_s3_data,
+    save_to_s3,
+    save_json_dict,
+)
+from dap_prinz_green_jobs import BUCKET_NAME, PROJECT_DIR, get_yaml_config, logger
+
+from ojd_daps_skills.pipeline.extract_skills.extract_skills import (
+    ExtractSkills,
+)  # import the module
+
```

Comment:
I think you can get rid of all the extract_skills_library_path paramater input bits and find it automatically using:
```
import ojd_daps_skills
extract_skills_library_path  = ojd_daps_skills.__file__.split("/__init__.py")[0]
```